.github/workflows/ci-test.yml

```diff
@@ -18,7 +18,9 @@ jobs:
       with:
         k8sVersion: ${{ matrix.k8sVersion }}    
     - run: K8S_VERSION=${{ matrix.k8sVersion }} make ci-test
-#    - uses: shogo82148/actions-goveralls@v1
-#      if: env.K8S_VERSION == '1.25.x'
-#      with:
-#        path-to-profile: coverage.out
+    - name: Send coverage
+      # should only send converage once https://docs.coveralls.io/parallel-builds
+      if: matrix.k8sVersion == '1.27.x'
```

Comment:
Why only `1.27.x`? 

Just since we don't need coverage more than once?

If so would like a note calling out this reasoning.

Reply:
Yes. It is called out in the comment. The only reason I picked 1.27 is that that's the latest of the ones we have been testing extensively.